### ISSUE TITLE
Inconsistent naming of norwegian translations on Transifex and git

### ISSUE THREAD

**Issue by petterreinholdtsen** [mysociety/alaveteli#2046]
The Norwegian bokmål translation is named nb_NO, while Norwegian nynorsk is named nn.  It would make more sense if they were named nb and nn (or nb_NO and nn_NO, but I recommend against it to stay consistent with most free software translated to Norwegian).

Can Transifex and git be updated to use nb and nn, not nb_NO and nn?

**Comment by garethrees** [comment]
For reference, Transifex [supports](https://www.transifex.com/languages):

- Norwegian Bokmål (nb)
- Norwegian Bokmål (Norway) (nb_NO)
- Norwegian (Norway) (no_NO)
- Norwegian Nynorsk (nn)
- Norwegian Nynorsk (Norway) (nn_NO)

**Comment by garethrees** [comment]
```irc
[09:45:05]  <gareth>	 pere so just to check my understanding, the action here is to rename `locale/nb_NO` to `locale/nb`?
[09:46:45]  <pere>	 yes.  and change it in transifex too.
```

**Comment by garethrees** [comment]
```irc
[10:03:27]  <pere>	 btw, on <URL: https://alaveteli-dev.nuug.no/ >, where do the string 'Norsk bokmål' come from?  I suspect it would be better to just write 'Bokmål'.
[10:06:40]  <gareth>	 pere https://github.com/mysociety/alaveteli/blob/rails-3-develop/lib/languages.rb#L114
```

**Comment by petterreinholdtsen** [comment]
You might find
<URL: http://people.skolelinux.org/pere/blog/Spr_kkoder_for_POSIX_locale_i_Norge.html >
useful as background information.  It is only in Norwegian, sorry about
that.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by garethrees** [comment]
@petterreinholdtsen I've made you a Coordinator on the `nb` locale for Alaveteli https://www.transifex.com/organization/mysociety/team/33529/members/nb/

**Comment by garethrees** [comment]
Could you confirm that the changes in this PR and on Transifex meet your needs; if so, I'll get it merged. We'll leave removing `nb_NO` until Release 0.21 (#2049).

**Comment by petterreinholdtsen** [comment]
[Gareth Rees]
> Could you confirm that the changes in this PR and on Transifex meet
> your needs; if so, I'll get it merged. We'll leave removing `nb_NO`
> until Release 0.21 (#2049).

The changes look good to me.  We will use the locale/nb/app.po from git,
and continue the translation of nb on Transifex.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by garethrees** [comment]
Merged as of https://github.com/mysociety/alaveteli/commit/bd1c11a86f2647753ac52a037385a829edd6a56c

These will be released to master in #2049
